src/sagas/filtersAside.js

```diff
@@ -62,14 +63,14 @@ export function* fetchStat(action) {
 
     const responses = yield all(requests);
 
-    const { data: dataCU } = responses.shift();
-    const dataC            = countC ? responses.shift()?.data : {};
-    const dataL            = countL ? responses.shift()?.data : {};
+    const { data: dataCU }                  = responses.shift();
+    const { data: { locations, ...dataC } } = countC ? responses.shift() : { data: false };
```

Comment:
what if responses.shift().data is not defined?

Reply:
was not think that its possible, its same number as responses that was inserted
add validation?